doc/source/testing.rst

```diff
@@ -45,34 +39,54 @@ prior to merging or submitting a pull request.
 
     $ tox
 
+List available scenarios
+------------------------
+
+List all available scenarios. This is useful to target specific Python and
+Ansible version for the functional and unit tests.
+
+    $ tox -l
+
+
 Unit
 ----
 
 Run all unit tests with coverage.
 
 .. code-block:: bash
 
-    $ tox -t unit
+    $ tox -e 'py{27,35,36,37}-ansible{25,26,27}-unit'
```

Comment:
Oh... you probably misunderstood my point. Just do this:
```suggestion
    $ tox -e unit
```
and it should select all envs matching `unit` factor

Reply:
Or maybe not... Let me check something

Reply:
Disregard my comments, I confused how it works. All's fine.